dev-scripts/mtlscerts.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+
+# very much just copied from https://lemariva.com/blog/2019/12/portainer-managing-docker-engine-remotely
+# production use should involve a real external certificate management system
+
+export CURRENT=$(pwd)
+export HOST=portainer.p1.alho.st
+export CERTDIR=~/.config/portainer/certs/
```

Comment:
maybe recieve that as a parameter? 
```suggestion
export CERTDIR=${1:-"~/.config/portainer/certs/"}
```